code/__DEFINES/clothing.dm

```diff
@@ -19,3 +19,8 @@
 #define HAS_SENSORS 1
 /// Suit sensor is present and is forced to display information (used on prisoner jumpsuits)
 #define LOCKED_SENSORS 2
+
+/// Wrapper for adding clothing based traits
+#define ADD_CLOTHING_TRAIT(mob, trait) ADD_TRAIT(mob, trait, "[CLOTHING_TRAIT]_[REF(src)]")
+/// Wrapper for removing clothing based traits
+#define REMOVE_CLOTHING_TRAIT(mob, trait) REMOVE_TRAIT(mob, trait, "[CLOTHING_TRAIT]_[REF(src)]")
```

Comment:
I kind of dislike the use of context specific variables inside of global defines as they hide what the define is actually doing and make it so you can't call it from outside of a context where src is the item you are applying it to. It could easilly lead to some bugs or confusion if used wrong. Instead it would be better if the clothing that it should apply to was passed in as a parameter and that was used instead of src.